eawx-build/Exceptions/NoRelativePathException.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+namespace EawXBuild.Exceptions {
+    public class NoRelativePathException : Exception {
```

Comment:
Custom exceptions should always implement the following three base constructors (see [Microsoft's Docs](https://docs.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions)), it also should have the `[Serializable]` attribute:
```suggestion
[Serializable]
public class NoRelativePathException : Exception
{
    public NoRelativePathException()
    {
    }

    public NoRelativePathException(string message)
        : base(message)
    {
    }

    public NoRelativePathException(string message, Exception inner)
        : base(message, inner)
    {
    }
}
```

Reply:
For that kind of boilerplate code I usually use the following template:

```csharp
$HEADER$using System;
using System.Runtime.Serialization;
namespace $NAMESPACE$
{
  [Serializable]
  public class $CLASS$ : Exception {
	public $CLASS$()
    {
    }

    public $CLASS$(string message) : base(message)
    {
    }

    public $CLASS$(string message, Exception innerException) : base(message, innerException)
    {
    }

    protected $CLASS$(SerializationInfo info, StreamingContext context) : base(info, context)
    {
    }
  }
}
```